drivers/can/can_mcux_flexcan.c

```diff
@@ -14,6 +14,10 @@
 #include <sys/byteorder.h>
 #include <fsl_flexcan.h>
 
+#if defined(CONFIG_NET_SOCKETS_CAN)
+#include "socket_can_generic.h"
+#endif /* CONFIG_NET_SOCKETS_CAN */
```

Comment:
How is this change related to imx rt?

Reply:
this is part of an existing feature. I moved it back to the socket can area.